dedibox-network/ipv6/how-to/configure-ipv6-linux.mdx

```diff
@@ -17,127 +17,66 @@ categories:
   - You have an account and are logged into the [Dedibox console](https://console.online.net)
   - You have [created](/dedibox/dedicated-servers/how-to/order-a-server) a Dedibox dedicated server
   - You have [requested](/dedibox-network/ipv6/how-to/request-prefix) a /48 IPv6 prefix
-  - You have configured the [DHCPv6 client](/dedibox-network/ipv6/how-to/configure-dhcpv6)
 </Message>
 
-<Message type="important">
-  * Make sure you have configured the [DHCPv6 client](/dedibox-network/ipv6/how-to/configure-dhcpv6) on your system before setting up the network configuration.
-  * Make sure [IPv6 SLAAC](/dedibox-network/ipv6/how-to/) is enabled on servers supporting this feature.
-</Message>
-
-## How to configure IPv6 on Debian
-
-1. Open the network configuration file `/etc/network/interfaces` in a text editor and edit it as follows:
-    ```
-    auto eno1
-    iface eno1 inet6 static
-        address IPV6ADDRESS
-        netmask PREFIXLENGTH
-    ```
 
-    Remember to replace `eno1` with the proper internet interface name.
-2. Restart the network:
-    ```
-    sudo systemctl restart networking.service
-    ```
+`systemd-networkd` is the network manager of [systemd](https://en.wikipedia.org/wiki/Systemd). It is provided natively on most Linux distributions and can be used for the configuration of IPv6 with its DHCPv6 client feature.
 
 <Message type="tip">
-  The network interface is initialized with the command `allow-hotplug` by default on Debian 9. It is possible that the network restart fails with this configuration. In this case, you can initialize the network with `auto` to avoid the problem.
+  When specifying the IP address of your Dedibox, you need to enter it in [CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing). The netmask for the main IPv4 of a Dedibox server is `/24`, the netmask for a failover IPv4 is `/32`. The netmask for the IPv6 is `/64`.
 </Message>
 
-### How to configure IPv6 on Debian without systemd
-
-If you do not want to use **systemd** to start your services, use the following configuration.
-
-1. Open the network configuration file `/etc/network/interfaces` in a text editor and edit it as follows:
-    ```
-    iface eno1 inet6 static
-        pre-up modprobe ipv6
-        address IPV6ADDRESS
-        netmask PREFIXLEN
-        post-up dhclient -cf /etc/dhcp/dhclient6.conf -6 -P -d -v $IFACE
-    ```
-
-    Adapt the interface name (`eno1`) to your needs, as well as the IPv6 address and the netmask.
-2. Restart the network:
+1. Find the name of your public Internet interface using the `ip link show` command. In this example, the name of the interface is `eno1`. Take a note of it, as it will be required in the next steps.
```

Comment:
```suggestion
1. Run the `ip link show` command to find the name of your public internet interface. Make sure to store it, as it will be required in the next steps.
    <Message type="note">
      In this example, the name of the interface is `eno1`.
    </Message>
```

Reply:
Suggestion applied in local git but I would advocate for "Internet" to remain capitalized.